### ISSUE TITLE
it only sync local files to google drive but can't sync files from google drive to local pi folder

### ISSUE THREAD

**Issue by jja67** [pageauc/rclone4pi#4]
I just installed this but could only sync files from the local files on pi to the google drive. but when I tried to sync the files on my google drive from the server to the local folder on the Pi, it couldn't work.

**Comment by pageauc** [comment]
That is right, script clone command is setup to sync in one direction. You
would need to use other clone parameters to do a Two way sync or to sync
from gdrive folder to raspberry pi

See clone help by running rclone command with no parameters or check rlone
web site https://rclone.org. and see docs section

Claude

**Comment by pageauc** [comment]
Also check rclone web site commands section for details and examples but be
careful testing since you could easily loose data if not done right
